idr/marshal1.go

```diff
@@ -0,0 +1,54 @@
+package idr
+
+import (
+	"fmt"
+
+	"github.com/jf-tech/go-corelib/jsons"
+)
+
+// j1NodePtrName returns a categorized name for a *Node pointer used in JSONify1
+func j1NodePtrName(n *Node) string {
+	if n == nil {
+		return "(nil)"
```

Comment:
What's the context we use `(nil)` here? I mean it looks a bit strange to me. Is that possible we use something like `*string` and return nil?